river/proba/gaussian.py

```diff
@@ -286,13 +285,17 @@ def __call__(self, x):
 
     def cdf(self, x):
         x = list(x.values())
-        return multivariate_normal(self.mu, self.var, allow_singular=True).cdf(x)
+        return multivariate_normal([*self.mu.values()], self.var, allow_singular=True).cdf(x)
 
     def sample(self):
```

Comment:
This method should return a dictionary :)

Reply:
Right, consistency is the key. :) Done.